src/docs/patterns/tables.mdx

```diff
@@ -0,0 +1,463 @@
+    ]}
+    captionsUnderImages={['', 'DONT:']}
+/>
+
+## Property Name
+
+In tables, you would call out properties of each data entry only once in the header row. In lists, however, either the values are clear enough on their own without explanations, or property names have to be included in every list item.
+
+<ImageGrid
+    images={[ImplicitValueTable]}
+    captionsUnderImages={[`The values are explained by the table header row.`]}
+    gridContainerProps={{ style: { justifyContent: 'flex-start' } }}
+/>
+<ImageGrid
+    images={[ImplicitValueList1, ImplicitValueList2]}
+    captionsUnderImages={[
+        `DONT: It is not clear what those values in the subtitle mean.`,
+        `CAUTION: Be careful when calling out property names in lists, as they weaken the scanability.`,
+    ]}
+/>
+
+## Consumer-Facing Applications
+
+Avoid using tables in consumer-facing applications. Lists mentally match real-world individual objects, while tables can be more abstract. Use tables only if your users really need to examine technical details.
+
+<Divider />
+
+# Responsive
+
+In general, **we do not encourage tables on mobile platforms**, as table-related interactions tend to be complicated for small viewports, and mobile phones also often lack the computational power to process massive data set.
+
+<ImageGrid
+    images={[TableOnMobile]}
+    regularWidth
+    caption={`Users can only see a very small portion of the table through mobile devices.`}
+/>
+
+## Understanding Users' Goal
+
+<ImageGrid images={[UserInField]} regularWidth caption={`While on foot, it is very hard to examine small details.`} />
+
+Before you design a table structure for a mobile application, you should conduct user researches and have a clear understanding of users' intentions. When they land on this page, Do they desire to do complicated cross-comparison between multiple rows and columns? Is a full table necessary?
+
+For industrial applications, your users would often be using a mobile app when they are out in the field. They are paying more attention to their surroundings, so keep the mobile app interaction minimal.
+
+## Breaking Down Tables into Lists
+
+If a user only needs a few parameters to identify the correct data entry and make decisions, a list with a few selected data points would suffice.
+
+<ImageGrid
+    images={[HomePagePreview, TableStackedAsList]}
+    captionsUnderImages={[
+        `On a crowded page, show a preview of the table content, with the option to view the full table in a dedicated page.`,
+        `Stack each property into a list item, with the ability to click into a list item to view the full item details.`,
+    ]}
+/>
+
+## Pagination
+
+Pagination is especially helpful on hand-held devices to reduce the load of both the server and the client-side render.
```

Comment:
This is a little confusing...this is coming after the section where we just said you should not use tables on mobile, but now we are saying pagination is great for tables on mobile.